pkg/collector/corechecks/servicediscovery/module/impl_linux.go

```diff
@@ -85,6 +88,7 @@ func newDiscovery() *discovery {
 		config:             newConfig(),
 		mux:                &sync.RWMutex{},
 		cache:              make(map[int32]*serviceInfo),
+		ignorePids:         make(map[int32]bool, len(ignoreServices)),
```

Comment:
the initial size does not make sense
this would be listing pids, and the size of ignoreServices is constant

Reply:
why don't we add pids that report true on `shouldIgnoreComm`?

Reply:
I've added code to set `ignoredPid` is was found that `shouldIgnoreComm` is true